src/collectors/common-contexts/disk.size.h

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#ifndef NETDATA_DISK_SIZE_H
+#define NETDATA_DISK_SIZE_H
+
+#include "common-contexts.h"
+
+typedef struct {
+    RRDSET *st_avgsz;
+    RRDDIM *rd_avgsz_reads;
+    RRDDIM *rd_avgsz_writes;
+} ND_DISK_AVGSIZE;
+
+typedef struct {
+    RRDSET *st_uavgsz;
+    RRDDIM *rd_avgsz_bytes;
+} ND_DISK_UAVGSIZE;
+
+static inline void common_disk_avgsize(ND_DISK_AVGSIZE *d, const char *id, const char *name, uint64_t bytes_read, uint64_t bytes_write, uint64_t sector_size, int update_every, instance_labels_cb_t cb, void *data) {

```

Comment:
This chart is different now bytes/sec vs op/s sec now

Reply:
Hello @stelfrag ,

I think you wanted to say ["KiB/operation"](https://github.com/netdata/netdata/blob/0a6c06e11650c057004196a9196710efda18ae59/src/collectors/proc.plugin/proc_diskstats.c#L2012).

Or is there something I am missing?